requirements-torch-cuda.txt

```diff
@@ -1,10 +1,10 @@
 # Tensorflow cpu-only version.
-tf-nightly-cpu==2.16.0.dev20231109  # Pin a working nightly until rc0.
+tf-nightly-cpu==2.16.0.dev20231221  # Pin a working nightly until rc0.
 
 # Torch with cuda support.
 --extra-index-url https://download.pytorch.org/whl/cu118
-torch==2.1.0
-torchvision==0.16.0
+torch==2.1.2+cu118
```

Comment:
Is this is sync with our pinned torch versions for keras itself?

Reply:
Do you mean if this is the same version of torch used at keras-team/keras?

In that case, the version is currently not the same. [They're running torch 2.1.1](https://github.com/keras-team/keras/blob/master/requirements-torch-cuda.txt), since 2.1.2 was [released](https://pypi.org/project/torch/) Dec 14, and their last Dependabot run was Dec 1st.

Once Dependabot runs again (Jan 1) and both projects merge their respective PRs, the versions will always match.